src/components/Room/Room.tsx

```diff
@@ -41,6 +45,19 @@ export function Room({
     roomId
   )
 
+  useEffect(() => {
+    const fetchSound = async () => {
+      audioBufferContainer.current = await fetch(
+        process.env.PUBLIC_URL + '/sounds/new-message.aac'
+      )
+        .then((response: Response) => response.arrayBuffer())
+        .then((arrayBuffer: ArrayBuffer) =>
+          audioContext.decodeAudioData(arrayBuffer)
+        )
+    }
+    fetchSound().catch(console.error)
```

Comment:
I'd like to see us using a `try`/`catch` here instead of `catch()`. For both this and the `await`/`then` comment above, we can probably solve this by inducing `async` scope inside of the `useEffect`:

```ts
(async () {
  try {
    await fetch(...)
    ...
  } catch (e) {
    console.error(e)
  }
})()
```

Reply:
Okay, i push that